### DISCUSSION TITLE
some missing files in third party folder for 3.3 download

### DISCUSSION THREAD

**Discussion by osgwraiESC** [pelicanmapping/osgearth#2245] [Support Q&A]
just an fyi, lerc and rapidjson folders in third-party folder are empty when i unpack osgearth-osgearth-3.3.tar.gz, no biggy i just fetched from the mainline to make the build work (other wise you get a couple errors during make) ...

**Comment by jasonbeverage** [comment]
That's to be expected, we are using submodules to fetch that code right now to simplify building from source.
